From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Mon, 8 Nov 2021 10:47:56 +1000
Subject: hwdb: remove the tablet pad entry for the UC-Logic 1060N

Origin: upstream, https://github.com/systemd/systemd/commit/2d8840eb6632a43956b73a1042e8b8244623494a
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1926860

This entry only matches on vid/pid, so the pen event node of the device
would also get assigned the ID_INPUT_TABLET_PAD property - making it
break with libinput.

On top of that, UC-Logic's tablets re-use USB ids, so now we're breaking
multiple devices this way.

To get this device tagged correctly, use libwacom which has the
per-device hwdb entries.

Fixes #17953

This reverts commit 0fbe78ac7a4c5689660b1c1e2c956ffa1a2e4ee5
---
 hwdb.d/60-input-id.hwdb | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/hwdb.d/60-input-id.hwdb b/hwdb.d/60-input-id.hwdb
index 2b1aa45..c4101cc 100644
--- a/hwdb.d/60-input-id.hwdb
+++ b/hwdb.d/60-input-id.hwdb
@@ -56,11 +56,6 @@
 
 # Sort by brand, model
 
-# UC-Logic TABLET 1060N Pad
-id-input:modalias:input:b0003v5543p0081*
- ID_INPUT_TABLET=1
- ID_INPUT_TABLET_PAD=1
-
 # XP-PEN STAR 06
 id-input:modalias:input:b0003v28bdp0078*
  ID_INPUT_TABLET=1
